airflow/models.py

```diff
@@ -748,6 +752,71 @@ def __init__(self, dag):
         self.pickle_hash = hash(dag)
         self.pickle = dag
 
+class SchedulerState(Base, LoggingMixin):
+    """
+    scheduler_state stores the state of the scheduler. When state is RUNNING, all operators
+    are free to launch.  When state is PAUSING, no new operators are scheduled.  When state
+    is IDLE, only externally-running tasks are still active, meaning airflow can safely be
+    redeployed.
+    """
+
+    __tablename__ = "scheduler_state"
+
+    state = Column(String(50), primary_key=True)
```

Comment:
See previous comment about support for multiple scheduler instances. This class would need to have a scheduler ID of some sort.

Reply:
Documentation: we discussed offline and agreed that this change is not necessary.